docs/reference/troubleshoot.md

```diff
@@ -80,9 +68,13 @@ namespace. To get the current Crossplane logs, run the following:
 kubectl -n crossplane-system logs -lapp=crossplane
 ```
 
-Remember that much of Crossplane's functionality is provided by Stacks. You can
-use `kubectl logs` to view Stack logs too, though Stacks may not run in the
-`crossplane-system` namespace.
+Remember that much of Crossplane's functionality is provided by providers. You
+can use `kubectl logs` to view provider logs too.
+
+> Note that Crossplane emits few logs by default - events are typically the best
+> place to look for information about what Crossplane is doing. You may need to
+> restart Crossplane (or your provider) with the `--debug` flag if you can't 
```

Comment:
Is this possible? Won't package manager override the `Deployment`?

Reply:
@muvaf I believe a flag should persist as we only patch the deployment in the provider revision controller:

https://github.com/crossplane/crossplane/blob/59620600805c4d8297a9a696ddf46104a09371f5/internal/controller/pkg/revision/reconciler.go#L220

That being said, I would recommend that a user reference a `ControllerConfig` in this situation.

Reply:
@hasheddan users would need to add the flag to `command: ` array. Do we have this array right now in the default case? If we do, I believe patch will replace the whole list once it sees any difference.

Reply:
I think this note has value regardless of how difficult it may be to actually restart the controller if it's under package manager management.